### ISSUE TITLE
Deprecated warning in emysql.hrl

### ISSUE THREAD

**Issue by ghost** [Eonblast/Emysql#140]
I'm getting the following error when including the emysql.hrl file. However it compiles and works fine.

```
../libs/Emysql/include/emysql.hrl:37: Warning: type queue/0 is deprecated and will be removed in OTP 18.0; use use queue:queue/0 or preferably queue:queue/1
../libs/Emysql/include/emysql.hrl:38: Warning: type gb_tree/0 is deprecated and will be removed in OTP 18.0; use use gb_trees:tree/0 or preferably gb_trees:tree/2
../libs/Emysql/include/emysql.hrl:39: Warning: type queue/0 is deprecated and will be removed in OTP 18.0; use use queue:queue/0 or preferably queue:queue/1
```

**Comment by jlouis** [comment]
Indeed, but this is only present in Erlang 17.0+ and it will break on R16B03 and earlier if we fix it.

The warning is benign until we hit 18.0, so right now I am for waiting a bit. See also this if you want to patch it:

http://erlang.org/pipermail/erlang-questions/2014-March/078081.html

which is a result of a discussion I started because of the troubles these changes will create.

**Comment by joekinley** [comment]
Aaaaand I ran into it with Erlang 18. Any plans to fix this?

**Comment by jlouis** [comment]
Yes and no. If you provide a PR, someone may help you pull it in. I decided to leave maintenance of this project about 9 months ago, and there is not much activity on it directly. Perhaps development has moved to another place. If you find such a place, get back to me so I can start sending people in the right direction.

**Comment by derek121** [comment]
Try Inaka's fork: https://github.com/inaka/Emysql Worked for me.
